prov/sockets/src/sock_rma.c

```diff
@@ -262,7 +266,7 @@ ssize_t sock_ep_rma_writemsg(struct fid_ep *ep, const struct fi_msg_rma *msg,
 
 	SOCK_EP_SET_TX_OP_FLAGS(flags);
 	if (flags & SOCK_USE_OP_FLAGS)
-		flags |= tx_ctx->attr.op_flags;
+		flags |= op_flags;
```

Comment:
Oh yes, I thought tx/rx flags were separate for base/alias eps. So, we need to check the ep->op flags in tx/rx operations.